examples/gum/debug_symbol/src/main.rs

```diff
@@ -0,0 +1,21 @@
+use frida_gum::DebugSymbol;
+use frida_gum::{Gum, Module};
+use lazy_static::lazy_static;
+
+lazy_static! {
+    static ref GUM: Gum = unsafe { Gum::obtain() };
+}
+
+fn main() {
+    let symbol = Module::find_export_by_name(None, "mmap").unwrap();
+    let symbol_details = DebugSymbol::from_address(symbol).unwrap();
```

Comment:
Can you add a `DebugSymbol::from_name` as well? This way we can do `DebugSymbol::from_name("mmap")` like in GumJS

Reply:
Fixed in d0188a5ae248fc3702d23c44e05dc5164133a88c